contracts/database/orm/orm.go

```diff
@@ -125,6 +125,8 @@ type Query interface {
 	Where(query any, args ...any) Query
 	// WhereIn adds a "where column in" clause to the query.
 	WhereIn(column string, values []any) Query
+	// OrWhereIn adds an "or where column in" clause to the query.
+	OrWhereIn(column string, values []any) Query
```

Comment:
Good implementation, just one nitpick: please sort the method by the first letter. 👍

Reply:
Sure 🙂